### PR TITLE
[BUG] Clarify relationship between default risk score and severity levels

### LINKED ISSUE #1705: [DOCS]Default risk score Range for Critical should be 74 - 99 instead of 74 - 100
## Description 
Default risk score Range for Critical should be 74 - 99 instead of 74 - 100

## URL

https://www.elastic.co/guide/en/security/8.1/rules-ui-create.html#rule-ui-basic-params

## Screen-Shoot

- Actual Result

image

- Expected Result

image

https://user-images.githubusercontent.com/59917825/157625216-9c815781-c9ee-422a-adf7-8bdb172de483.mp4

### PR DESCRIPTION
**Pull request by joepeeples**
Resolves #1705. The risk score range in step 1e (74-100) is still valid, because that step identifies general ranges that the user can set the **Default risk score** value to, not hard and fast values that are controlled by the **Default severity** level. This change helps explain the relationship between the two settings.

Also, per dev team, the risk score is being set to 99 by design, and the user can can still adjust the score higher or lower if they want.

Preview [here](https://security-docs_1722.docs-preview.app.elstc.co/guide/en/security/master/rules-ui-create.html#rule-ui-basic-params) (step #1e).

### CODE REVIEW

**Review by spong — APPROVED** [review]
LGTM! Thanks for the added clarification here @joepeeples! 🙂

File: docs/detections/rules-ui-create.asciidoc [hunk 1]
```diff
@@ -308,8 +308,7 @@ values:
 image::images/severity-mapping-ui.png[]
 +
 NOTE: For threshold rules, not all source event values can be used for overrides; only the fields that were aggregated over (the `Group by` fields) will contain data. Please also note that overrides are not supported for event correlation rules.
-.. *Default risk score*: A numerical value between 0 and 100 that correlates
-with the *Severity* level. General guidelines are:
+.. *Default risk score*: A numerical value between 0 and 100 that indicates the risk of events detected by the rule. This setting is set to a default value when you change the *Severity* level, but you can also adjust the risk score as needed. General guidelines are:
```
Comment (nastasha-solomon) [inline docs/detections/rules-ui-create.asciidoc:311]:
```suggestion
.. *Default risk score*: A numerical value between 0 and 100 that indicates the risk of events detected by the rule. This setting is set to a default value when you change the *Severity* level, but you can adjust the risk score as needed. General guidelines are:
```

**Review by nastasha-solomon — APPROVED** [review]
Super minor comment, but otherwise LGTM!

File: docs/detections/rules-ui-create.asciidoc [hunk 2]
```diff
@@ -308,8 +308,7 @@ values:
 image::images/severity-mapping-ui.png[]
 +
 NOTE: For threshold rules, not all source event values can be used for overrides; only the fields that were aggregated over (the `Group by` fields) will contain data. Please also note that overrides are not supported for event correlation rules.
-.. *Default risk score*: A numerical value between 0 and 100 that correlates
-with the *Severity* level. General guidelines are:
+.. *Default risk score*: A numerical value between 0 and 100 that indicates the risk of events detected by the rule. This setting is set to a default value when you change the *Severity* level, but you can adjust the risk score as needed. General guidelines are:
```
Comment (benironside) [inline docs/detections/rules-ui-create.asciidoc:311]:
```suggestion
.. *Default risk score*: A numerical value between 0 and 100 that indicates the risk of events detected by the rule. It defaults to a preset when you change the *Severity* level, but you can adjust it as needed. General guidelines are:
```

**Review by benironside — COMMENTED** [review]
Left one suggestion but LGTM

**Review by karanbirsingh-qasource — APPROVED** [review]
Approved and thanks for sharing additional comment about the risk details range

**Pull request merged**